src/aspire/reconstruction/kernel.py

```diff
@@ -130,13 +131,15 @@ def convolve_volume_matrix(self, x):
         # Order is important here.  It's about 20% faster to run from 1 through 6 compared with 6 through 1.
         # TODO: Experiment with scipy order; try overwrite_x argument
         for i in range(6):
-            x = fft(x, N_ker, i, overwrite_x=True)
+            _pad_width = [(0, 0)] * 6
+            _pad_width[i] = (0, N_ker)
```

Comment:
Shouldn't this be `N_ker-N`?

Reply:
Yes, you are right, I dropped it messing around. Thanks! Fixing.